dom0-updates/qubes-dom0-update

```diff
@@ -222,75 +220,77 @@ echo "Using $UPDATEVM as UpdateVM to download updates for Dom0; this may take so
 
 # qvm-run by default auto-starts the VM if not running
 readonly dom0_updates_dir=/var/lib/qubes/dom0-updates
-qvm-run --nogui -q -u root -- "$UPDATEVM" "mkdir -m 775 -p -- '$dom0_updates_dir'" || exit 1
-qvm-run --nogui -q -u root -- "$UPDATEVM" "chown -R -- user:user '$dom0_updates_dir'" || exit 1
-qvm-run --nogui -q -- "$UPDATEVM" "rm -rf -- '$dom0_updates_dir/etc' '$dom0_updates_dir/var/lib/rpm'" || exit 1
-(cd / && exec tar -c -- var/lib/rpm etc/yum/vars etc/yum.repos.d etc/yum.conf etc/dnf/dnf.conf etc/pki/rpm-gpg/RPM-GPG-KEY-* 2>/dev/null) |
```

Comment:
You are reverting 37be9fbe610c67069ad4e59b595cc56a51da0514, I don't think it stopped being relevant (even if default install doesn't use it).